doc/man3/BIO_should_retry.pod

```diff
@@ -117,8 +117,8 @@ BIO_should_read(), BIO_should_write(), BIO_should_io_special(), and
 BIO_should_retry() return either true or false based on the actual conditions
 of the B<BIO>.
 
-BIO_retry_type() returns a mask of the cause of a retry condition or false if
-there is no retry condition.
+BIO_retry_type() returns a flag combination present the cause of a retry condition
```

Comment:
In such case "flag combination present*ing* the cause"... Though "representing" might be better...

Reply:
Fixed